lib/xpm_ruby/schema/staff/add.rb

```diff
@@ -0,0 +1,16 @@
+require_relative "../../types"
+
+module XpmRuby
+  module Schema
+    module Staff
+      Add = Types::Hash.schema(
+        Name: Types::String,
+        Address?: Types::String,
```

Comment:
Is address list a single text field, not separated by line, city, etc?

As usual their API documentation doesn't actually give you a useful example... :smile:

<img width="349" alt="image" src="https://user-images.githubusercontent.com/645930/115165251-5e529e00-a0f0-11eb-9004-eb2c5ea1be61.png">

Reply:
yeah, their docs are...interesting  😄